<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>473</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>C:/Users/Ayaz/.designer/backup/VU_top bar logo.png</selectedon>
   </iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{
background-color:rgb(255,255,255);
}
QPlainTextEdit {
padding: 2px;
border-style: solid;
border: 1px solid gray;
border-radius: 1px;
}
QPlainTextEdit {
    background-color:rgb(255, 255, 255);color:#rgb(0, 0, 0);border:1px;border-style:solid;border-color:#999;
}
QComboBox{
background-color:rgb(255, 255, 255);
padding: 2px;
border-style: solid;
border: 1px solid gray;
border-radius: 1px;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">Qwidget{
 
border: 3px solid gray;
border-radius: 40px;
background: white;
}</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>501</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
background-color:rgb(255,255,255)
}
QLabel{
background-color:rgb(255,255,255)
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="ui_label_txt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string>logo</string>
     </property>
    </widget>
    <widget class="QLabel" name="ClassName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_text">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Name">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>320</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 
                text-align: center;
                border-radius: 20px;
                 background-color:rgb(240, 228, 230)


}

QPushButton:hover {
    background-color:rgb(211, 207, 255);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
     </property>
     <property name="text">
      <string>submit</string>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_txt_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tags">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="upload">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 
                text-align: center;
                border-radius: 20px;
                 background-color:rgb(240, 228, 230)


}

QPushButton:hover {
    background-color:rgb(211, 207, 255);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
     </property>
     <property name="text">
      <string>upload</string>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="file_">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="description">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="name_check">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tags_list">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tags_check">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>220</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 54,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>320</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 
                text-align: center;
                border-radius: 20px;
                 background-color:rgb(240, 228, 230)


}

QPushButton:hover {
    background-color:rgb(211, 207, 255);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="getlist">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
 
                text-align: center;
                border-radius: 20px;
                 background-color:rgb(240, 228, 230)


}

QPushButton:hover {
    background-color:rgb(211, 207, 255);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
     </property>
     <property name="text">
      <string>Get list</string>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>20</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
